Assets/MixedRealityToolkit-Examples/Demos/StandardShader/Materials/ShaderBallClippingPlaneInner.mat

```diff
@@ -4,12 +4,12 @@
 Material:
   serializedVersion: 6
   m_ObjectHideFlags: 0
-  m_PrefabParentObject: {fileID: 0}
+  m_CorrespondingSourceObject: {fileID: 0}
```

Comment:
Please revert changes to these materials, we should not include auto-updated files by Unity. They need to be tested in the scope of a Standard Shader PR